src/emulators-ui.css

```diff
@@ -11,7 +11,9 @@
 }
 
 .emulator-canvas, .emulator-video {
-    image-rendering: "crisp-edges";
+    image-rendering: -moz-crisp-edges;
```

Comment:
Since this hasn't been part of js-dos I have been using:

```css
image-rendering: pixelated;
image-rendering: crisp-edges;
```

Looks like Firefox has supported `crisp-edges` since v65. Also Safari has supported `crisp-edges` since v6.1, and if you wanted to support below that I think it's `-webkit-optimize-contrast`.

Based on the `browserslist` [entry](https://github.com/js-dos/emulators-ui/blob/main/package.json#L7), I think we aren't expected to support that low.

Reply:
`-moz-crisp-edges` is required by Firefox 90 for Android. 

`-webkit-crisp-edges` is necessary for Safari from 6.1 to 9.3, but now I see that this rule is not applied to canvas, that's why it can be removed.

I will add unprefixed `crisp-edges` before `pixelated` because the latter is widely supported and it was not intended to be used with smoothing algorithms. 

Do you agree?

Reply:
Done.